app/src/main/java/woowacourse/movie/activity/MainActivity.kt

```diff
@@ -52,26 +53,24 @@ class MainActivity : AppCompatActivity() {
         if (ContextCompat.checkSelfPermission(
                 this,
                 Manifest.permission.POST_NOTIFICATIONS,
-            ) != PackageManager.PERMISSION_GRANTED
+            ) == PackageManager.PERMISSION_GRANTED
         ) {
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
-                if (shouldShowRequestPermissionRationale(Manifest.permission.POST_NOTIFICATIONS)) {
-                    // 권한 요청 거부한 경우
-                } else {
-                    requestPermissionLauncher.launch(Manifest.permission.POST_NOTIFICATIONS)
-                }
-            } else {
-                // 안드로이드 12 이하는 Notification에 관한 권한 필요 없음
+            return
+        }
+        when {
+            Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU && shouldShowRequestPermissionRationale(
+                Manifest.permission.POST_NOTIFICATIONS,
+            ) -> {
+                Toast.makeText(this, getString(R.string.check_notification), Toast.LENGTH_LONG).show()
+            }
+            Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU -> {
+                requestPermissionLauncher.launch(Manifest.permission.POST_NOTIFICATIONS)
             }
+            else -> {}
```

Comment:
```suggestion
            else -> Unit
```
구현부가 없는 경우에는 Unit을 활용할 수 있어요